.github/workflows/screener-run.yml

```diff
@@ -1,32 +1,44 @@
 name: Screener run
-
 on:
-  workflow_run:
-    workflows:
-      - Screener build
-    types:
-      - completed
+  pull_request:
 
+  #workflow_run:
+  #  workflows:
+  #    - Screener build
+  #  types:
+  #    - completed
 env:
   AZURE_STORAGE_CONNECTION_STRING: ${{secrets.AZURE_STORAGE_CONNECTION_STRING}}
-
 jobs:
   screener-react-northstar:
     runs-on: 'ubuntu-latest'
-
     name: Screener @fluentui/react-northstar
-
     steps:
       - uses: actions/checkout@v3
         with:
-          ref: ${{github.event.workflow_run.head_branch}}
           fetch-depth: 0
 
       - uses: actions/setup-node@v3
         with:
           node-version: 14.18.1
           cache: 'yarn'
 
+      - name: Check if test app artifact deployed
+        uses: actions/github-script@v6
+        id: skip-screener
+        with:
+          script: |
+            const core = require('@actions/core');
```

Comment:
It seems that `core` import is not required, https://github.com/actions/github-script#actionsgithub-script

Reply:
right, removing it